wsd/DocumentBroker.cpp

```diff
@@ -2625,6 +2625,22 @@ void DocumentBroker::onViewLoaded(const std::shared_ptr<ClientSession>& session)
     //TODO: Take the WOPI lock, if necessary.
 }
 
+std::shared_ptr<ClientSession> DocumentBroker::getFirstAuthorizedSession() const
```

Comment:
I presume there is a plan to build on getFirstAuthorizedSession() later, doesn't seem used in this pr

Reply:
Yes, patches forthcoming!